result.rb

```diff
@@ -0,0 +1,2 @@
+[1] pry(main)> data
+=> [{:name=>"2259A3648041", :balance=>"924,14", :currency=>"MDL"}, {:name=>"2259A36117121", :balance=>"101,78", :currency=>"MDL"}]
```

Comment:
output should be in JSON format. it's quite easy to convert hash to json. read about it